### PR TITLE
Minor GSG tweaks

### PR DESCRIPTION
**Pull request by carlescufi**
- Add ARMv8-R
- Add a table for the required dependencies versions

### CODE REVIEW

**Comment by carlescufi** [comment]
image

**Review by gmarull — APPROVED** [review]

**Review by nashif — APPROVED** [review]

File: doc/getting_started/index.rst [hunk 1]
```diff
@@ -68,33 +85,15 @@ Next, you'll install some host dependencies using your package manager.
...
-
-               sudo apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main'
-
-         c) Then install the updated version of cmake with ``apt``:
-
-            .. code-block:: bash
-
-               sudo apt update
-               sudo apt install cmake
+         And check those against the table in the beginning of this section.
```
Comment (ru-fu) [inline doc/getting_started/index.rst:94]:
```suggestion
         Check those against the versions in the table in the beginning of this section.
```

File: doc/getting_started/index.rst [hunk 2]
```diff
@@ -68,33 +85,15 @@ Next, you'll install some host dependencies using your package manager.
...
+         Refer to :ref:`installation_linux` page for additional info on updating
```
Comment (ru-fu) [inline doc/getting_started/index.rst:95]:
```suggestion
         Refer to the :ref:`installation_linux` page for additional information on updating
```

**Review by ru-fu — COMMENTED** [review]
Some nitpicking.

**Review by ru-fu — APPROVED** [review]

**Pull request merged**